ReactNativeClient/lib/components/global-style.js

```diff
@@ -24,6 +24,8 @@ const globalStyle = {
 	raisedColor: '#003363',
 	raisedHighlightedColor: '#ffffff',
 
+	barStyle: 'dark-content',
```

Comment:
Each theme has an `appearance` property which will tell you if it's "light" or "dark, which is currently used to set the `keyboardAppearance` property. Likewise please could you make the `barStyle` property derive from the `appearance` property. Please have a look at how `keyboardAppearance` is set. I expect in your case you just need to add one line `style.barStyle = style.appearance + "-content"`

Reply:
The `barstyle` actually has to be the opposite of whatever the `appearance` value is set to. `'dark-content'` makes the status bar icons black. And `'light-content'` makes them white.
So I've done this instead: 

`style.barStyle = ((style.appearance === 'dark') ? 'light-content' : 'dark-content')`

I can break that into a normal if statement if you prefer.

Reply:
Yes this is a good fix, but looks like you didn't commit the changes?

Reply:
Yeah, sorry about that. I can't seem to get the latest version to run in the android emulator. The version I have in my branch seems to work, but I wanted to make sure it worked in the latest version. It's a pretty small change, so I wouldn't mind if a new pull request was made or something else.